hieroplatform/src/test/java/org/hiero/sketch/IntConverter.java

```diff
@@ -0,0 +1,13 @@
+package org.hiero.sketch;
```

Comment:
I need to put some converter for the histogram. If I put a null instead I get a null pointer exception at ArrayColumn.asDouble(). It is triggered by this line:
double val = column.asDouble(currRow, converter);
in the createHistogram method of the histogram object.